src/spikeinterface/core/base.py

```diff
@@ -79,6 +79,18 @@ def __init__(self, main_ids: Sequence) -> None:
         # preferred context for multiprocessing
         self._preferred_mp_context = None
 
+    @property
+    def name(self):
+        return self._annotations.get("name", self.__class__.__name__)
```

Comment:
@h-mayorquin this now handles the retrieval of either a custom name or the class name by default

Reply:
OK.